src/main/java/io/reactivex/Completable.java

```diff
@@ -760,8 +857,11 @@ public final Completable endWith(CompletableConsumable other) {
      * @throws NullPointerException if onAfterComplete is null
      */
     @SchedulerSupport(SchedulerSupport.NONE)
+    @Deprecated
```

Comment:
There will be an API pass, until then no API is to be deleted that has a 1.x counterpart (would mess up the diff anyway).